Contest_API.md

```diff
@@ -1480,6 +1480,43 @@ Returned data:
 ]
 ```
 
+### Commentary
+
+Commentary on events happening in the contest
+
+The following endpoints are associated with commentary:
+
+| Endpoint                         | Mime-type        | Required? | Source @WF | Description                                                                    |
+| -------------------------------- | ---------------- | --------- | ---------- | ------------------------------------------------------------------------------ |
+| `/contests/<id>/commentary`      | application/json | no        | not used   | JSON array of all commentary with elements as defined in the table below       |
+| `/contests/<id>/commentary/<id>` | application/json | no        | not used   | JSON object of a single commentary with elements as defined in the table below |
+
+JSON elements of award objects:
+
+| Name          | Type        | Required? | Nullable? | Source @WF | Description |
+| ------------- | ----------- | --------- | --------- | ---------- | ----------- |
+| id            | ID          | yes       | no        | not used   | identifier of the commentary. |
+| time          | TIME        | yes       | no        | not used   | time of the commentary message. |
+| contest\_time | RELTIME     | yes       | no        | not used   | contest time of the commentary message. |  
+| message       | string      | yes       | no        | not used   | commentary message text. May contain special tags for [teams](#teams) and [problems](#problems) on the format `#t<team ID>` and `#p<problem ID>` respectively.|
+| team\_ids     | array of ID | yes       | yes       | not used   | JSON array of [team](#teams) IDs the message is related to.
+| problem\_ids  | array of ID | yes       | yes       | not used   | JSON array of [problem](#problems) IDs the message is related to.
+
+#### Example
+
+Request:
+
+` GET https://example.com/api/contests/wf14/commentary`
+
+Returned data:
+
+```json
+[{"id":"143730", "time":"2021-03-06T19:02:02.328+00", "contest_time":"0:02:02.328", "message": "#t20 made a submission for #anttyping. If correct, they will solve the first problem and take the lead", "team_ids": ["314089"], "problem_ids": ["anttyping"]}, 
```

Comment:
Missing `p` for problem tags? Or is `#anttyping` not a problem?

Reply:
Yes, that was a mistake. Thanks for catching. Updated now.